database/seeders/CreateInitialAdminAccount.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Database\Seeders;
+
+use App\Models\User;
+use Illuminate\Database\Seeder;
+
+class CreateInitialAdminAccount extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        // unguard(): disable all mass assignable restrictions.
+        User::unguard();
+
+        User::create([
+            'email' => 'bui.minh.dung@sun-asterisk.com',
+            'password' => bcrypt('Admin123.'),
+            'first_name' => 'Administrator',
+            'last_name' => 'Account',
+            'is_active' => true,
+            'username' => 'administrator@account',
+            'phone' => '0986653341',
+            'email_verified_at' => now(),
+            'address' => 'Developer',
+        ]);
+
+        $user = User::find(1);
```

Comment:
```suggestion
        $user = User::where('email', ...)->first();
```

nếu query tạo user ko phải id = 1 thì sao?